src/arch/Sound/RageSoundDriver.cpp

```diff
@@ -17,34 +31,40 @@ RageSoundDriver *RageSoundDriver::Create( const RString& drivers )
 	std::vector<RString> drivers_to_try;
```

Comment:
It's kinda weird how `drivers_to_try` is weirdly cased throughout these files. Let's make them the same with proper naming style.